addons/stock/models/product.py

```diff
@@ -762,6 +779,11 @@ class ProductTemplate(models.Model):
         string="Tracking", required=True, default='none', # Not having a default value here causes issues when migrating.
         compute='_compute_tracking', store=True, readonly=False, precompute=True,
         help="Ensure the traceability of a storable product in your warehouse.")
+    serial_sequence_id = fields.Many2one(
+        'ir.sequence', 'Serial/Lot Numbers Sequence', compute='_compute_serial_sequence_id', store=True, precompute=True,
+        help='Technical Field: The Ir.Sequence record that is used to generate serial/lot numbers for this product')
+    serial_prefix_format = fields.Char('Custom Lot/Serial', help=SERIAL_PREFIX_FORMAT_HELP_TEXT)
+    next_serial = fields.Char(compute='_compute_next_serial')
```

Comment:
I'm not sure it should be that way. For me it would be better to have sequence_id stored and serial_prefix_format that is an inverse/compute

Reply:
I would use the naming lot_sequence_id since we use lot everywhere else in the code